pysixdesk/lib/study.py

```diff
@@ -898,20 +856,16 @@ def fun(elem):
                 flag = li.set(iden, outputs['task_id'][i], rec)
                 if flag:
                     break
-            else:
-                res = SpecialDict.fromkeys(rec, iden_names)
-                res.set(iden, outputs['task_id'][i], rec)
-                group_list.append(res)
+                else:
```

Comment:
The `else` clause is with `for` statement!

Reply:
Oh yes indeed my bad ! I didn't know having an `else` after a `for` that was possible.